CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# v1.2.0.RC1 2018-08-09
+
+## Changes since last release
+
+* Support for single values.
+  [Documentation](https://github.com/samvera-labs/valkyrie/wiki/Using-Types#singular-values)
+* Optimistic Locking.
+  [Documentation](https://github.com/samvera-labs/valkyrie/wiki/Optimistic-Locking)
+* Remove reliance on ActiveFedora for Fedora Storage Adapter.
+* Only load adapters if referenced.
+* Postgres Adapter uses transactions for `save_all`
+* Resources now include `id` attribute by default.
+
+## Special Thanks
+
+This release was made possible by a community sprint undertaken between Penn
+State University Libraries & Princeton University Library. Thanks to the
+following participants who made it happen:
+
+* [awead](https://github.com/awead)
+* [cam156](https://github.com/cam156)
+* [escowles](https://github.com/escowles)
+* [hackmastera](https://github.com/hackmastera)
+* [jrgriffiniii](https://github.com/jrgriffiniii)
+* [mtribone](https://github.com/mtribone)
+* [tpendragon](https://github.com/tpendragon)
+
```

Comment:
Add Dan

Reply:
This is what I get for looking at commits.